game_event/views.py

```diff
@@ -1 +1,37 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from .models import GameEvent
+from datetime import date
+
+
+def game_events(request):
+    game_events = GameEvent.objects.all()
+
+    if request.GET.get('min_players'):
+        try:
+            min_players = int(request.GET.get('min_players'))
+            if min_players > 0:
+                game_events = game_events.filter(min_number_of_players__gte=min_players)
+        except ValueError:
+            pass
+
+    if request.GET.get('max_players'):
+        try:
+            max_players = int(request.GET.get('max_players'))
+            if max_players > 0:
+                game_events = game_events.filter(max_number_of_players__lte=max_players)
+        except ValueError:
+            pass
+
+    if request.GET.get('max_level'):
+        try:
+            max_level = int(request.GET.get('max_level'))
+            if max_level > 0:
```

Comment:
I think the max_level should be 0 (included) to 11 (not included),
like the fields on class Rating.

Reply:
@MaayanMashhadi can this one be resolved?

Reply:
> @MaayanMashhadi can this one be resolved?

Yes, but unfortunately I didn't success to resolve this conversation.